### ISSUE TITLE
Allow the user to define a page offset so that :<number> commands align with document defined page numbers.

### ISSUE THREAD

**Issue by sebastinas** [pwmt/zathura#120]
*On GitLab by @cjayross on Apr 29, 2019, 21:05*

---

My most often use case for Zathura is for reading books with a large number of pages and most of them don't have any metadata for accessing via Zathura's ToC indexing.
A feature that I have known would be very useful for me is if Zathura gave me the option to set a "page offset" integer that would be added automatically to the numbers I use to jump to a page (using the `:<number>` command) and would be stored in the document's config file so that it will persist between instances.

For example, if a given book begins indexing it's pages on the PDF's 17th page (so on page 17, the index is 1; on 18 it's 2, and so on), it would be great if I could use command like `:set_page_offset 16` so that from then on a call such as `:245` will bring me to the 261st page of the PDF rather than the 245th.
Often times this offset will be small, meaning that losing the ability to properly jump to the PDF's first `page_offset` pages won't be much loss in functionality for me since I would simply be using the `gg` macro instead.

**Comment by sebastinas** [comment]
*On GitLab by @cjayross on Apr 29, 2019, 21:56*

---

I've looked into the source code and it appears that there is this functionality already available, however, the `:offset <number>` command doesn't seem to change any effects of the goto shortcut.
It seems that this might actually be a bug.

Edit: I have found that the `<number>G` shortcut functions with the offset correctly. Perhaps then the `:<number>` syntax is used to override the offset as a feature.

The documentation should be updated to reflect this.
